### DISCUSSION TITLE
Window Sharing From Taskbar

### DISCUSSION THREAD

**Discussion by patel8786** [valinet/ExplorerPatcher#1658] [Ideas]
Hi

It seems that window sharing directly from the taskbar is not supported when taskbar style is "windows 10" even if the setting is enabled in windows.

<img width="238" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/17146116/75d24fbd-c16c-4277-848a-358daedf862a">

Is this something that can be added?

Many thanks

**Comment by pyrates999** [comment]
Is this for sharing in microsoft teams?

**Comment by patel8786** [comment]
Yep

**Comment by pyrates999** [comment]
Only if the code for doing this is part of the windows 10 taskbar code, it might be able to be enabled.  But if it is not, just like widgets are not part of the windows 10 taskbar, it can't be enabled.

**Comment by emufan** [comment]
I would love to see this enabled as well.

**Comment by Amrsatrio** [comment]
Later this year. (I hope)
